File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
     int count  = 0;
     for(int i=0; i<nums.length; i++){
        int next = (i+1)%nums.length;
        if(nums[i] > nums[next]){
            count++;
        }
     }
     return count <= 1;
    }
}

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.py
class Solution(object):
    def check(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        count = 0
        for i in range(0, len(nums)):
            next = (i+1)%len(nums)
            if nums[i] > nums[next]:
                count +=1
        
        return count <= 1